commands/utils/depsutil_test.go

```diff
@@ -62,7 +62,7 @@ func createRemoteRepo(t *testing.T, project string, server *config.ServerDetails
 	createRemoteRepoServices.ArtDetails = rtDetails
 	var repoKey string
 	switch project {
-	case "npm", "yarn":
+	case "npm", "yarn2":
```

Comment:
what about yarn1?

Reply:
this tests checks repo creating in artifactory, since there is no 'yarn' repo in artifactory (only npm) so there is no difference between the yarn versions for this test so I used one of the repos of yarn